### PR TITLE
Messages

### PR DESCRIPTION
**Pull request by lamara**

### CODE REVIEW

**Comment by dlongley** [comment]
Needs rebasing.

**Comment by mattcollier** [comment]
Rebased.

File: lib/auth.js [hunk 1]
```diff
@@ -96,7 +96,8 @@ bedrock.events.on('bedrock.init', function(callback) {
       jsigs.sign(didDocument, {
         algorithm: 'LinkedDataSignature2015',
         privateKeyPem: bedrock.config.idp.owner.privateKey,
-        creator: didDocument.publicKey[0].id
+        creator: didDocument.publicKey[0].id,
+        algorithm: 'LinkedDataSignature2015'
```
Comment (davidlehn) [inline lib/auth.js]: Algorithm is so nice you set it twice.

File: lib/auth.js [hunk 2]
```diff
@@ -817,7 +818,8 @@ api.signJsonLdAsIdentityService = function(obj, options, callback) {
       var _options = {
         algorithm: 'LinkedDataSignature2015',
         privateKeyPem: privateKey,
-        creator: publicKey.id
+        creator: publicKey.id,
+        algorithm: 'LinkedDataSignature2015'
```
Comment (davidlehn) [inline lib/auth.js]: Algorithm is so nice you set it twice.

**Pull request merged**